src/SDK/Language/Android.php

```diff
@@ -47,207 +44,193 @@ public function getFiles()
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/models/UploadProgress.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/models/UploadProgress.kt.twig',
-                'minify'        => false,
             ],
             [
                 'scope'         => 'default',
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/WebAuthComponent.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/WebAuthComponent.kt.twig',
-                'minify'        => false,
             ],
             [
                 'scope'         => 'default',
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/KeepAliveService.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/KeepAliveService.kt.twig',
-                'minify'        => false,
             ],
             [
                 'scope'         => 'default',
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/views/CallbackActivity.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/views/CallbackActivity.kt.twig',
-                'minify'        => false,
             ],
             [
                 'scope'         => 'default',
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/services/Service.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/services/Service.kt.twig',
-                'minify'        => false,
             ],
             [
                 'scope'         => 'default',
                 'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/services/Realtime.kt',
                 'template'      => '/android/library/src/main/java/io/appwrite/services/Realtime.kt.twig',
-                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/cookies/CookieStoreExtensions.kt',
+                'template'      => '/android/library/src/main/java/io/appwrite/cookies/CookieStoreExtensions.kt.twig',
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/cookies/InMemoryCookieStore.kt',
+                'template'      => '/android/library/src/main/java/io/appwrite/cookies/InMemoryCookieStore.kt.twig',
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/cookies/SharedPreferencesCookieStore.kt',
+                'template'      => '/android/library/src/main/java/io/appwrite/cookies/SharedPreferencesCookieStore.kt.twig',
```

Comment:
Do you think a better naming convention is to have

io/appwrite/cookie
  - store
      - Extensions.kt
      - InMemory.kt
      - SharedPreference.kt 
  - Internal.kt 


WDYT @abnegate  ?

Reply:
@christyjacob4 I think the folder structure provides a better separation of concerns but the naming makes it hard to tell what the classes are doing without reading some code.

I have updated with just the directory changes, does that work for you?

Reply:
Understood and your change makes sense 👍  Thank you